Lib/email/message.py

```diff
@@ -287,7 +287,7 @@ def get_payload(self, i=None, decode=False):
             raise TypeError('Expected list, got %s' % type(self._payload))
         payload = self._payload
         # cte might be a Header, so for now stringify it.
-        cte = str(self.get('content-transfer-encoding', '')).lower()
+        cte = str(self.get('content-transfer-encoding', '')).lower().strip(' ')
```

Comment:
Why the explicit `' '`? [str.strip](https://docs.python.org/3/library/stdtypes.html#str.strip) defaults to stripping whitespace.

```suggestion
        cte = str(self.get('content-transfer-encoding', '')).lower().strip()
```